src/services/asset.ts

```diff
@@ -13,3 +13,21 @@ export function getAsset(userId: number, typeId: number): Promise<IAsset> {
 
   return request;
 }
+
+export function uploadAsset(params: any): Promise<IAsset> {
+  const { assetType, files } = params;
+  console.log(params);
+  const formData = new FormData();
+  formData.append('file', files[0]);
+
+  const fetch = new Fetch();
```

Comment:
Fetch is an abstract class - you dont need to create new instance of it.
just use Fetch.request(***)